active_learning/active_learning.py

```diff
@@ -1,21 +1,29 @@
-from util.datastructures import MetaPath
+from util.datastructures import MetaPath, MetaPathRatingGraph
 
 # mock data
 mock_paths = [MetaPath([1, 2], [3]), MetaPath([1, 2, 1], [3, 4]), MetaPath([1, 3, 2], [1, 1]),
               MetaPath([1, 3, 2], [1, 1]), MetaPath([1, 3, 2], [1, 1])]
 
 # algorithm types
-NO_RATING = 'no_rating'
+NO_USER_FEEDBACK = 'no_user_feedback'
 BASELINE = 'baseline'
 
 
 class Oracle:
     """
     Abstract class for the Oracle the Active Learner interacts with. 
     """
+    number_of_calls = None
```

Comment:
Comment missing. What calls does this member count?

Reply:
the number of calls :)